src/utils.ts

```diff
@@ -0,0 +1,78 @@
+import * as core from '@actions/core'
+
+interface FieldValueType {
+  [key: string]: string
+}
+
+interface OptionType {
+  id: string
+  name: string
+}
+
+interface ProjectFieldsType {
+  options: OptionType[]
+  name: string
+  dataType: string
+  id: string
+}
+
+/**
+ * Returns an array of field updates for the GraphQL API
+ 
+ * @param fieldValues
+ * @param projectFields
+ * @returns {Promise<[]>}
+*/
+
+export async function getFieldUpdates(fieldValues: {[key: string]: string}, projectFields: []) {
+  const array: ProjectFieldsType[] = projectFields
```

Comment:
Why do we need this `const` definition here? Can't we use `projectFields`?

Reply:
we need ro declare the properties of the projectfields, or else we get an error of property name does not exist